packages/router/src/adapters/redisCache/index.ts

```diff
@@ -0,0 +1,28 @@
+import RedisClient, { Redis } from "ioredis";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import RedisMock from "ioredis-mock";
+
+import { Cache } from "../../lib/entities";
+import { getContext } from "../../router";
+
+import { getOutstandingLiquidity, storeOutstandingLiquidity, removeOutstandingLiquidity } from "./cache";
+
+let redis: Redis;
+
+export const getRedis = (): Redis => redis;
+
+export const getCache = async (): Promise<Cache> => {
+  const { config } = getContext();
+  if (config.redisUrl) {
+    redis = new RedisClient(config.redisUrl);
+  } else {
+    redis = new RedisMock();
+  }
```

Comment:
There are some commands / features that the redis mock hasn't yet implemented. Think well need to test this with high rates of spammy quotes to see how the mock performs before saying okay to this as a solution

Reply:
We are using the most highly used, basic Redis commands which are most definitely the first ones that they have implemented so I'm not worried if our functionality is there. I agree we need to load test with spammy quotes, but I don't see the connection between spammy quotes and the mock.

Reply:
It's just a note about the use of the mock in prod in general for future compatibility/ functionality

Reply:
We definitely aren't using the mock in prod, all deployments will definitely have full Redis instances.

Reply:
Ah got it, I misunderstood at the standup